### PR TITLE
Fix: Fixing formating and linting across the website code

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**What kind of change does this PR introduce?**

Formats the code files.

**Issue Number:**
#539 

**Screenshots/videos:**
N/A

**If relevant, did you update the documentation?**

N/A

**Summary**
Code files are now formatted

### CODE REVIEW

**Comment by utnim2** [comment]
In my opinion we should add  a script here that enforces linting and check on every PR through github actions, 
And mentioned it in the contribuding guide
In this way we dont have to do this manually all the time, the actions will fail if the changes pushed does not have linting

**Comment by utnim2** [comment]
In my opinion we should add  a script here that enforces linting and check on every new PR created through github actions, 
And mentioned it in the contribuding guide
In this way we dont have to do this manually all the time, the actions will fail if the changes pushed does not have linting

**Comment by frouioui** [comment]
I think we should do what @utnim2 mentioned

**Comment by DhairyaMajmudar** [comment]
Sure will work on this 💯

**Comment by DhairyaMajmudar** [comment]
@frouioui I have added the script `npm run format` using which the files can be formatted. Also updated the readme.md file : )

**Comment by DhairyaMajmudar** [comment]
> To come back on what @utnim2 was saying, I think we should make this a CI check on every PR just like what we have in `.github/workflows/golang_lint.yaml`.

Sure looking forward to add lint CI

File: .github/workflows/typescript_lint.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: PR Workflow
```
Comment (frouioui) [inline .github/workflows/typescript_lint.yaml:1]: The name of the workflow should be more descriptive 😄

File: .github/workflows/typescript_lint.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - "website/**"
+
+jobs:
+  linting_and_type-checking:
+    name: Linting, Formatting and Type checking
```
Comment (frouioui) [inline .github/workflows/typescript_lint.yaml:11]: The job name as well, we should mention it is related to react/ts

File: .github/workflows/typescript_lint.yaml (hunk 1 above)
Comment (DhairyaMajmudar) [reply .github/workflows/typescript_lint.yaml]: Yeah good catch : D 

I have improved this in my recent commit.

File: .github/workflows/typescript_lint.yaml (hunk 2 above)
Comment (DhairyaMajmudar) [reply .github/workflows/typescript_lint.yaml]: Done this one also : )

**Comment by DhairyaMajmudar** [comment]
Moving changes to PR https://github.com/vitessio/arewefastyet/pull/545

**Pull request closed without merging**